mm2src/coins/test_coin.rs

```diff
@@ -78,7 +78,11 @@ impl MarketCoinOps for TestCoin {
         unimplemented!()
     }
 
-    fn tx_enum_from_bytes(&self, bytes: &[u8]) -> Result<TransactionEnum, String> { unimplemented!() }
+    fn tx_enum_from_bytes(&self, _bytes: &[u8]) -> Result<TransactionEnum, MmError<TransactionErr>> {
+        MmError::err(TransactionErr::Plain(
+            "tx_enum_from_bytes is not supported for Test coin yet.".to_string(),
+        ))
+    }
```

Comment:
We shouldn't be afraid to panic on `TestCoin` methods

Reply:
It's not because I wanted to avoid using panic here. It's because I wanted implement same thing for all unimplemented methods.